test/azul_test_case.py

```diff
@@ -316,9 +323,13 @@ def dummy_get_credentials(_self):
             # These must match what `moto` uses to mock the instance metadata
             # response (see InstanceMetadataResponse.metadata_response() in
             # moto.instance_metadata.responses).
-            return Credentials(access_key='test-key',
-                               secret_key='test-secret-key',
-                               token='test-session-token')
+            credentials = Credentials(access_key='test-key',
+                                      secret_key='test-secret-key',
+                                      token='test-session-token')
+            # Since we used Credentials and not a subclass of Credentials, these
+            # credentials lack an expiration date, so we add some in here.
```

Comment:
Which subclass(es) _do(es)_ define an expiry time? And _why_ do we add one here?

```suggestion
            # Since we use the Credentials class itself rather than a subclass,
            # this instance lack an expiration date, so we add one.
```